django_project/certification/migrations/0014_auto_20171116_1326.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
Its not clear to me why we even have this migration? All you have changed is the help text (which not does not comply with coding convention used elsewhere). Just remove this migration.